docs/source/en/using-diffusers/schedulers.md

```diff
@@ -165,6 +165,62 @@ image
 
 Most images look very similar and are comparable in quality. Again, it often comes down to your specific use case so a good approach is to run multiple different schedulers and compare the results.
 
+### Custom Timestep Schedules
```

Comment:
I added a section here @stevhliu

Reply:
we should probably refactor later and also in the same place, talk about `use_karras_sigmas` and `rescale_zero_terminal_snr`
most of the scheduler configs you just should keep the same as the default scheduler, but these are the configurations that the user can play around with

Reply:
for now, feel free to give it a review :)